sdk/clientcore/pom.xml

```diff
@@ -14,6 +14,8 @@
     <module>http-okhttp3</module>
     <module>optional-dependency-tests</module>
     <module>http-stress</module>
+    <module>annotation-processor</module>
+    <module>annotation-processor-tests</module>
     <module>tools</module>
```

Comment:
Do we have anything in `tools`? Can this be removed?

Reply:
The OpenRewrite PR is progressing in parallel. To avoid merge conflicts, I'll clean up any unnecessary items in the `tools` package afterwards if they aren't needed.